### PR TITLE
(#1142) - Make EObytes use Bytes for all operations

### PR DESCRIPTION
**Pull request by andreoss**
Per #1142

### CODE REVIEW

**Comment by yegor256** [comment]
@mximp take a look

File: eo-runtime/src/test/eo/org/eolang/bytes-tests.eo [hunk 1]
```diff
@@ -374,11 +374,11 @@
       $.equal-to
         "hello world"
 
-[] > bytes-hashes-equals
-  assert-that > @
-    05-5E-78.as-hash
-    $.equal-to
-      05-5E-78.as-hash
+# [] > bytes-hashes-equals
```
Comment (mximp) [inline eo-runtime/src/test/eo/org/eolang/bytes-tests.eo:377]: @andreoss do we need this commented code?

File: eo-runtime/src/test/eo/org/eolang/string-tests.eo [hunk 2]
```diff
@@ -110,11 +110,11 @@
     """
     $.equal-to "a\n b\n  c"
 
-[] > similar-strings-generate-similar-hashes
-  assert-that > @
-    "hello".as-hash
-    $.equal-to
-      "hello".as-hash
+# [] > similar-strings-generate-similar-hashes
```
Comment (mximp) [inline eo-runtime/src/test/eo/org/eolang/string-tests.eo:113]: @andreoss same here: do we need commented code?

File: eo-runtime/src/test/eo/org/eolang/bytes-tests.eo (hunk 1 above)
Comment (andreoss) [reply eo-runtime/src/test/eo/org/eolang/bytes-tests.eo:377]: @mximp There is a todo to reimplement `as-hash` and enable these two tests

Comment (mximp) [reply eo-runtime/src/test/eo/org/eolang/bytes-tests.eo:377]: @andreoss sorry, missed that was related. got it.

**Review by mximp — APPROVED** [review]

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@andreoss please, ping me in the messages when a PR is ready for another merge. Otherwise, I don't know when it's ready.

**Comment by andreoss** [comment]
@yegor256 it's ready

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**